calico_containers/pycalico/netns.py

```diff
@@ -51,6 +51,20 @@ def setup_logging(logfile, level=logging.INFO):
     _log.addHandler(handler)
 
 
+def increment_metrics(namespace):
+    """
+    Increment the metric of any default route that has a metric of 0.
```

Comment:
I thought @robbrockbank's design was to increment the metric of every default route if any have a metric of 0.  E.g. 
default via A metric 0
default via B metric 1

becomes

default via A metric 1
default via B metric 2

No?